cms/envs/bok_choy.py

```diff
@@ -147,7 +147,8 @@
 FEATURES['ENABLE_LIBRARY_INDEX'] = True
 FEATURES['ENABLE_CONTENT_LIBRARY_INDEX'] = False
 
-FEATURES['ORGANIZATIONS_APP'] = True
+ORGANIZATIONS_AUTOCREATE = False
```

Comment:
Is this `False` here because we don't want to have to mess with old Bok Choy tests?

Reply:
(not judgin', just curious)

Reply:
@davidjoy Yeah, I wanted to keep the behavior identical to avoid messing with any bok choy tests.

And in terms of the current usage of bok choy in edx-platform (a11y tests to make sure critical edx.org features are usable and accessible) I think it makes sense to use the same Organizations config that we use in Production.